docs/csharp/language-reference/preprocessor-directives.md

```diff
@@ -0,0 +1,428 @@
+---
+description: "Learn the different C# preprocessor directives that control conditional compilation, warnings, nullable analysis, and more"
+title: "C# preprocessor directives"
+ms.date: 03/17/2021
+f1_keywords: 
+  - "cs.preprocessor"
```

Comment:
f1 keywords from the deleted articles are now dropped.

Reply:
Thanks @Youssef1313 That's a great catch. I'll add them to the new page today.